src/components/index/RoadMapSection.tsx

```diff
@@ -0,0 +1,154 @@
+import { Container } from "@components/commons/Container";
+import { SectionTitle } from "@components/commons/SectionTitle";
+import { LeftArrow } from "@components/icons/assets/roadmap/LeftArrow";
+import { RightArrow } from "@components/icons/assets/roadmap/RightArrow";
+import classNames from "classnames";
+import { useTranslation } from "next-i18next";
+import { CSSProperties, ReactEventHandler } from "react";
+import Slider from "react-slick";
+
+export function YearAheadRoadMapSection(): JSX.Element {
+  const { t } = useTranslation("page-index");
+
+  return (
+    <>
+      <Container>
+        <SectionTitle
+          text={t("YearAheadRoadMapSection.label")}
+          customStyle="lg:w-fit md:w-[409px] w-[272px]"
+        />
+        <div className="lg:text-[52px] lg:leading-[52px] text-[32px] leading-[36px] text-dark-1000 pt-5 pb-6">
+          {t("YearAheadRoadMapSection.title")}
+        </div>
+      </Container>
+      <RoadMapSlider />
+    </>
+  );
+}
+
+interface RoadMap {
+  title: string;
+  timeline: string;
+  description: string;
+  image: string;
+}
+
+function RoadMapSlider(): JSX.Element {
+  const { t } = useTranslation("page-index");
+  const cards: RoadMap[] = t("YearAheadRoadMapSection.cards", {
+    returnObjects: true,
+  });
+  const settings = {
+    customPaging: () => (
+      <a>
+        <div className="w-full h-0.5" />
+      </a>
+    ),
+    dots: true,
+    dotsClass: "roadmap-dots",
+    infinite: false,
+    speed: 300,
+    slidesToShow: 1,
+    centerPadding: "0px",
+    lazyload: "progressive",
+    variableWidth: true,
+    swipeToSlide: false,
+    draggable: false,
+    // prevArrow: <PrevArrow />, // TODO: uncomment when more than 3 cards in road map
+    // nextArrow: <NextArrow />,
+    responsive: [
+      {
+        breakpoint: 1024,
+        settings: {
+          draggable: true,
+          swipeToSlide: true,
+        },
+      },
+      {
+        breakpoint: 600,
+        settings: {
+          centerMode: true,
```

Comment:
should the centerMode be false ?

Reply:
added this so that the slide is horizontally centered in mobile view